frontend/test/pytest/test_quantum_control.py

```diff
@@ -59,7 +59,7 @@ def pennylane_workflow(*args):
         else:
             return quantum_func(*args, ctrl_fn=PL_ctrl)
 
-    assert_allclose(catalyst_workflow(*args), pennylane_workflow(*args))
+    assert_allclose(catalyst_workflow(*args), pennylane_workflow(*args), atol=1e-7)
```

Comment:
I chatted about this on slack, but it is likely something to do with lightning and how it is compiled. It is not a big difference. Instead of 0 we get a value 1e-17

Reply:
> Instead of 0 we get a value 1e-17
That should be within the standard tolerance no? here we have e-7